npm-react-codemod/transforms/utils/ReactUtils.js

```diff
@@ -31,6 +31,7 @@ module.exports = function(j) {
       .filter(j.filters.VariableDeclarator.requiresModule(module))
       .size() === 1;
 
+  // TODO: any specific addon?
```

Comment:
cc @cpojer who can probably answer this. I'm going to guess we don't want to change anything.